cilium/cmd/helpers.go

```diff
@@ -406,3 +424,10 @@ func getIpv6EnableStatus() bool {
 	// returning the EnableIPv6 default status
 	return defaults.EnableIPv6
 }
+
+func mergeMaps(m1, m2 map[string]interface{}) map[string]interface{} {
+	for k, v := range m1 {
+		m2[k] = v
+	}
+	return m2
+}
```

Comment:
This is actually modifying the 2nd argument. I think you might want to create a new map and return that instead, i.e. `m3`